src/platform/mhv/downtime/components/MHVDown.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { formatDatetime, formatElapsedHours } from '../utils/date';
+
+function MHVDown({
+  appTitle = 'some of our health tools',
+  endTime,
+  startTime,
+}) {
+  const startString = startTime ? formatDatetime(startTime) : '';
+  const endString = endTime ? formatDatetime(endTime) : '';
+  const timeInterval = formatElapsedHours(startTime, endTime);
```

Comment:
Do you need a conditional here to check is any of the times do not exist?

Reply:
Yeah, this component gets the Date/Moment from the `DowntimeNotification` component which gets it from the vets-api maintenance window endpoint (via redux actions/reducers), so the chance of invalid times is minimal, but could happen.

When adding the `timeInterval` I was more focused on the fact that the [deprecated momentjs library](https://momentjs.com) is used by `DowntimeNotification`, so moment date objects should probably make an appearance as well. Wes created [a ticket to remove momentjs]( https://github.com/department-of-veterans-affairs/va.gov-team/issues/75609) fwiw

Reply:
Refactored this after adding in logic to check that the values exist, and are Date instances, or if they are moment instances that can be converted to Date instances.

Anyway, moved the date logic to the container component, `MHVDowntime` rather than repeat logic in the two child components.